h/static/scripts/annotator/plugin/toolbar.coffee

```diff
@@ -55,7 +66,17 @@ class Annotator.Plugin.Toolbar extends Annotator.Plugin
         "click": (event) =>
           event.preventDefault()
           event.stopPropagation()
-          @annotator.onAdderClick target: dataset: action: "comment"
```

Comment:
You should either not change this line or follow the same convention in the highlight button. It's oddly inconsistent the way you've done it.